src/common-events/utils/headless-cms/service.tsx

```diff
@@ -94,7 +94,7 @@ export const getAllPages = async (): Promise<PageInfo[]> => {
         locale: node.language.code.toLowerCase(),
         slug: node.slug,
       });
-      node.translations?.forEach((translation) => {
+      node.translations?.forEach((translation: PageType['translation']) => {
```

Comment:
This is a change that the github CI whined me about during the last merge process. I think this should be avoided and should not be added if not needed. Did the CI now request it?

Reply:
yes, build failed

Reply:
shouldn't this actually be PageType['translation'][number] or just a PageType? 

My change from the last PR: https://github.com/City-of-Helsinki/hobbies-helsinki/pull/58/files#diff-1874e43653b7f8b433c6b6a6eff0f44487d1f45df14794ab82607d7230a130dfR51 and https://github.com/City-of-Helsinki/hobbies-helsinki/pull/58/files#diff-366bb48be14ed35d071f776ae7f410400f9869da28a6fd82efb84d89f5c129c0R32